awx/main/dispatch/worker/task.py

```diff
@@ -85,6 +86,7 @@ def perform_work(self, body):
             'task': u'awx.main.tasks.RunProjectUpdate'
         }
         '''
+        settings.__clean_on_fork__()
```

Comment:
In reality, we should only need to call this *once* in the forked child process, and this function is what `multiprocessing.Process`'s target is pointed at.